examples/cifar/train_cifar_custom_loop.py

```diff
@@ -102,9 +109,7 @@ def main():
             optimizer.lr *= 0.5
             print('Reducing learning rate to: {}'.format(optimizer.lr))
 
-        x_array, t_array = convert.concat_examples(batch, args.gpu)
-        x = chainer.Variable(x_array)
-        t = chainer.Variable(t_array)
```

Comment:
Just to be clear, this is just a nonessential code improvement, right?

Reply:
Yes. (Actually, I thought that it is necessary changes, but I found it unnecessary after I send this PR.)